tox.ini

```diff
@@ -13,3 +13,5 @@ ignore=
     E731,
     # no longer best practice:
     W503
+    ; module level import not at top of file
+    E402,
```

Comment:
Might be able to do this with inline comments rather than globally.

Reply:
Was coping the Cylc `tox.ini`. We don't generally do this, but when we do we usually have our reasons. It's not a fault I think we'll miss.